test/vegeta/pacers.go

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// pacers contains self-defined pacers that can be used in Attack of Vegeta (for performance testing)
+
+package vegeta
```

Comment:
`vegeta` will collide with the actual package when this actually gets used.  Can you call this package `pacers` and split each into it's own file?

Reply:
Done.